src/cli.ts

```diff
@@ -187,22 +187,27 @@ function transformMarkdown(buf: Buffer) {
   const source = buf.toString()
   const output = []
   let state = 'root'
+  let sequence = ''
+  let match = ''
   let prevLineIsEmpty = true
   for (let line of source.split('\n')) {
     switch (state) {
       case 'root':
         if (/^( {4}|\t)/.test(line) && prevLineIsEmpty) {
           output.push(line)
           state = 'tab'
-        } else if (/^```(js|javascript)$/.test(line)) {
+        } else if (/^(```+|~~~+)(js|javascript)$/.test(line)) {
           output.push('')
           state = 'js'
-        } else if (/^```(sh|bash)$/.test(line)) {
+          sequence = line.match(/^(```+|~~~+)(js|javascript)$/)![1]
```

Comment:
This regexp can be split into a separate variable.

Reply:
I will reflect on this review next time.